### DISCUSSION TITLE
How to deploy to non root folder on IIS in production?

### DISCUSSION THREAD

**Discussion by HjalteParner** [jasontaylordev/CleanArchitecture#594] [General]
When deploying the solution to a website root folder, everything works as expected. However when deploying to a non root folder i.e. as a web application on IIS, all api calls goes to the website root i.e. /api instead of /webapp/api which causes all api calls to fail.

How do I solve this, please?

P.S. I have set the angular base-ref to the /webapp/ and added a web.config redirecting all request to /webapp/ as described as a solution many places to this issue. However the api calls still do not work.

**Comment by ayhamallan** [comment]
@HjalteParner  any chance that you got it solved ?
